<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>315</y>
     <width>201</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="imap_server_line_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imap_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>240</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройки IMAP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imap_server_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сервер:</string>
   </property>
  </widget>
  <widget class="QLabel" name="imap_port_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Порт:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mail_server_combo_box">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Yandex</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gmail</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mail.ru</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="imap_password_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QLabel" name="imap_login_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="imap_password_line_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="imap_login_line_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="smtp_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>240</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройки SMTP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="smtp_port_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Порт:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="smtp_password_line_edit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="smtp_server_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сервер:</string>
   </property>
  </widget>
  <widget class="QLabel" name="smtp_login_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>160</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="smtp_login_line_edit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="smtp_password_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="smtp_server_line_edit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="autoupdate_spin_box">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>86400</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoupdate_check_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Автообновление:</string>
   </property>
  </widget>
  <widget class="QLabel" name="seconds_label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>285</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>с.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mail_server_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>310</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Выставить настройки автоматически:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="imap_port_spin_box">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>993</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="smtp_port_spin_box">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>465</number>
   </property>
  </widget>
  <widget class="QComboBox" name="imap_auth_combo_box">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>START_TLS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="imap_auth_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Auth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="smtp_auth_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Auth:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="smtp_auth_combo_box">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>START_TLS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NONE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="smtp_name_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>280</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="smtp_name_line_edit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>mail_server_combo_box</tabstop>
  <tabstop>imap_server_line_edit</tabstop>
  <tabstop>imap_port_spin_box</tabstop>
  <tabstop>imap_login_line_edit</tabstop>
  <tabstop>imap_password_line_edit</tabstop>
  <tabstop>imap_auth_combo_box</tabstop>
  <tabstop>autoupdate_check_box</tabstop>
  <tabstop>autoupdate_spin_box</tabstop>
  <tabstop>smtp_server_line_edit</tabstop>
  <tabstop>smtp_port_spin_box</tabstop>
  <tabstop>smtp_login_line_edit</tabstop>
  <tabstop>smtp_password_line_edit</tabstop>
  <tabstop>smtp_auth_combo_box</tabstop>
  <tabstop>smtp_name_line_edit</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
